.github/pull_request_template.md

```diff
@@ -1,22 +1,32 @@
-<!--
-  Copy the task id from the bottom of the sidebar and paste it after FTRACK-
+<!--(
+  Copy the id and paste it to the appropriate CLICKUP-<id> / FTRACK-<id> /  SENTRY-<id> / ZENDESK-<id> link.
+  Please remember to remove the unused ones.
 -->
+Resolves : 
 
-Resolves FTRACK-
+* CLICKUP-
+* FTRACK-
+* SENTRY-
+* ZENDESK-
```

Comment:
I think it might be better to keep all of these as part of the comment above, since I believe it's more annoying having to remove the ones that are very rarely used every time, than to add the tracking issues you want to use.

Reply:
moved them into the comment, and added a line to express the need of adding one for reference